app/router.js

```diff
@@ -66,7 +66,10 @@ Router.map(function() {
     path: 'curriculum-inventory-reports'
   });
   this.route('curriculumInventoryReport', {
-    path: 'curriculum-inventory-reports/:curriculum_inventory_report_id'
+    path: 'curriculum-inventory-reports/:curriculum_inventory_report_id',
+    resetNamespace: true
```

Comment:
I don't understand `resetNamespace` in this context.  Seems like it results in the same url `/curriculum-inventory-reports/9/rollover` either way. I might be not understanding from the documentation what this actually does.